### PR TITLE
Itkovian log vmem requests

### PR DESCRIPTION
**Pull request by stdweird**
Replaces/continues #72

### CODE REVIEW

File: bin/myshowq.py [hunk 1]
```diff
@@ -114,26 +108,26 @@ def makemap(users, owner):
...
+    dest = "%s/.showq.pickle.map" % home
 
     if os.path.isfile(dest):
         try:
             execfile(dest)
         except Exception, err:
-            print "Failed to load map file %s: %s"%(dest,err)
-        if locals().has_key("map"):
+            print "Failed to load map file %s: %s" % (dest, err)
+        if "map" in locals:
```
Comment (wpoely86) [inline bin/myshowq.py]: `locals()` ?

File: lib/vsc/jobs/pbs/submitfilter.py [hunk 2]
```diff
@@ -43,22 +41,25 @@
 PBS_OPTION_REGEXP = re.compile(r"(?:^|\s)-(\w+)(?:(?!\s*(?:\s-\w+|$))\s+(.*?(?=\s*(?:\s-\w+|$))))?")
 
 # TODO: all lower and uppercase combos?
```
Comment (wpoely86) [inline lib/vsc/jobs/pbs/submitfilter.py]: Add `re.IGNORECASE` to the compile?

File: bin/submitfilter.py [hunk 3]
```diff
@@ -71,23 +71,32 @@ def make_new_header(sf):
...
+            make("-m", "n"),
         ])
 
-    #    vmem: add default when not specified
-    if not 'vmem' in state['l']:
-        (ppp, vpp) = get_cluster_mpp(state['_cluster'])
+    current_user = pwd.getpwuid(os.getuid()).pw_name
+
+    # vmem: add default when not specified
+    if 'vmem' not in state['l'] and 'pmem' not in state['l']:
```
Comment (wpoely86) [inline bin/submitfilter.py]: both `vmem` and `pmem` are used in several places. Make it a constant?

File: lib/vsc/jobs/pbs/submitfilter.py (hunk 2 above)
Comment (stdweird) [inline lib/vsc/jobs/pbs/submitfilter.py]: the uppercase ones would be invalid options

**Comment by stdweird** [comment]
@wpoely86 done

**Pull request merged**